velox/functions/remote/if/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+include(FBThriftCppLibrary)
+
+add_fbthrift_cpp_library(velox_remote_function_thrift RemoteFunction.thrift)
```

Comment:
Where is the thrift file you reference here committed?

Reply:
The thrift file is already committed, but currently the library is not being built using cmake (only meta's internal build). The file is here:

https://github.com/facebookincubator/velox/blob/main/velox/functions/remote/if/RemoteFunction.thrift